.github/workflows/test.yml

```diff
@@ -13,6 +13,26 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.10"]
+        test-case:
+          - name: Check Turtle syntax - docs/tern.ttl
+            command: ontotools file validate docs/tern.ttl
+          - name: Check Turtle syntax - docs/tern.shapes.ttl
+            command: ontotools file validate docs/tern.shapes.ttl
+          - name: Check Turtle syntax - docs/tern.ecoplots.shapes.ttl
+            command: ontotools file validate docs/tern.ecoplots.shapes.ttl
+          - name: Check Turtle syntax - docs/meta.shapes.ttl
+            command: ontotools file validate docs/meta.shapes.ttl
+          - name: Check Turtle syntax - docs/tern.profile.ttl
+            command: ontotools file validate docs/tern.profile.ttl
+          - name: Check Turtle syntax - docs/ecoplots.profile.ttl
+            command: ontotools file validate docs/ecoplots.profile.ttl
+
+          - name: Validate shapes with meta shapes - docs/tern.ttl
+            command: pyshacl -s docs/meta.shapes.ttl docs/tern.ttl
+          - name: Validate shapes with meta shapes - docs/tern.shapes.ttl
+            command: pyshacl -s docs/meta.shapes.ttl docs/tern.shapes.ttl
+          - name: Validate shapes with meta shapes - docs/tern.ecoplots.shapes.ttl
+            command: pyshacl -s docs/meta.shapes.ttl docs/tern.ecoplots.shapes.ttl
```

Comment:
Use `meta.shapes.ttl` to validate itself.

Reply:
Need to add `sh:name` and `sh:description` to the property shapes in `meta.shapes.ttl`.